docs/_posts/2019-03-05-go.md

```diff
@@ -0,0 +1,50 @@
+
+## … and beyond the toolchain 
+
+With Go, we can produce statically linked binaries that fully contain our applications and run them on different platforms. This capability supported our decision to move to Docker-based development for our building and testing.  
+
+Secretless is consumed as a Linux binary that is packaged inside a Docker image. That way, customers can easily get up and running once they grab our image. We also use Docker for our CI pipeline to build and publish our site, check styling and links, and run tests. We host these processes in Docker to ensure they receive the proper Go versioning for testing and to isolate their environments to ensure they always run in the same (and expected) way. 
+
+When it comes to testing, I have mixed feelings about Docker. On one hand, without it, we cannot properly test our code and on the other, Docker has added an extra layer of complexity that makes our project difficult to understand and unapproachable. We find Docker beneficial because the alternative is a large assortment of messy Bash scripts and boilerplate code. Furthermore, contributors no longer need to fuss with tooling in order to configure their environment correctly - all they need is Docker and they are ready to get rolling. But with this assumption, we have added an extra pre-requisite that ultimately limits those who can contribute. In terms of testing norms, Docker-based development has forced us to break the standard Go testing conventions. Our tests have become custom to our project, wrapping the known Go tests with Docker-specific code. Consequently, it may become initially unclear to contributors how our tests work.  
+
+To curb these complexities and ensure a positive contributor experience, we are working to retain the integrity of Go testing through simplification and trying to provide better documentation. We hope this will eventually give our contributors the support and guidance they need to get involved and continue to contribute to our project.  
+
+## Summing Up 
+
+Go is trending in the open source world and its support and community are only growing. For our project, we found Go’s native capabilities and supportive toolset to be specifically helpful as our project grew in complexity. Go’s sophistication and modern approach to computing are what have ultimately allowed us to stretch outside the conventions of a normal open source project.  
+
+Looking to contribute to [Secretless?](https://github.com/cyberark/secretless-broker) We would love nothing more!  
```

Comment:
maybe link to our CONTRIBUTING.md for contributing info?

Reply:
```suggestion
Looking to contribute to [Secretless?](https://github.com/cyberark/secretless-broker/blob/master/CONTRIBUTING.md) We would love nothing more!  
```

Reply:
good call! Adding to PR updates.